src/pages/_app.tsx

```
@@ -27,6 +27,8 @@ export default function App({ Component, pageProps }: AppProps) {
           {/* <a href="/settings">Settings</a> */}
           <a href="/scheduledOrders">Scheduled Orders</a>
           <a href="/scheduledProducts">Scheduled Products</a>
+          <a href="/storeSettings">Store Settings</a>
```

ESLint

[error] Do not use an `<a>` element to navigate to `/storeSettings/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)